_toc.yml

```diff
@@ -464,6 +464,8 @@ entries:
             title: Terminology
           - file: docs/products/postgresql/reference/idle-connections
             title: Idle connections
+          - file: docs/products/postgresql/reference/pg-long-running-queries
```

Comment:
Is this the right positioning? I would say this article is an howto with title "Detect and terminate long-running queries"

Reply:
Moved the file to `howto` section and changed the title to `Detect and terminate long-running queries`